disabled the use of ccache, to prevent FTBFS; as the compilation is done once
it has no backdraws to make the debian package.
Index: supercollider/CMakeLists.txt
===================================================================
--- supercollider.orig/CMakeLists.txt
+++ supercollider/CMakeLists.txt
@@ -148,11 +148,11 @@ add_definitions(-DBOOST_CHRONO_HEADER_ON
 # Detect CCache
 
 find_program(CCacheExectuable ccache)
-if( CCacheExectuable )
-  # only used with >=cmake-3.4
-  set( CMAKE_C_COMPILER_LAUNCHER   "${CCacheExectuable}" )
-  set( CMAKE_CXX_COMPILER_LAUNCHER "${CCacheExectuable}" )
-endif()
+#if( CCacheExectuable )
+#  # only used with >=cmake-3.4
+#  set( CMAKE_C_COMPILER_LAUNCHER   "${CCacheExectuable}" )
+#  set( CMAKE_CXX_COMPILER_LAUNCHER "${CCacheExectuable}" )
+#endif()
 
 #############################################
 # Options
